.nycrc

```diff
@@ -0,0 +1,6 @@
+{
+  "exclude": [
+    "**/internal/process/write-coverage.js"
```

Comment:
Is the wildcard necessary? It'll always be in lib/internal/process, unless the absolute path is used for comparison?

Reply:
@TimothyGu because the Makefile passes an absolute path for the test coverage's working directory:

`--temp-directory "/Users/benjamincoe/bcoe/node/.cov_tmp"`

nyc ends up working with the path:

`/Users/benjamincoe/bcoe/node/lib_/internal/process/write-coverage.js` when the file is processed by the exclude glob.

tldr; we definitely don't want to bake `/Users/benjamincoe/bcoe` into the exclude rule, and it seemed like prefixing with the `**` was the easiest solution.

Reply:
@bcoe Makes sense :+1: